bin/publish

```diff
@@ -29,10 +29,15 @@ if [ "$GIT_DESCRIPTION" = "v${VERSION}" ]; then
     docker push "$REGISTRY/$IMAGE_NAME:$tag"
   done
 
+  # Publish RedHat image to Internal Registry
+  docker tag "secrets-provider-for-k8s-redhat:${FULL_VERSION_TAG}" "$REGISTRY/${REDHAT_IMAGE}:${VERSION}"
+  docker push "$REGISTRY/${REDHAT_IMAGE}:${VERSION}"
```

Comment:
Should we just push it as secrets-provider-for-k8s-redhat instead f the full REDHAT_IMAGE name?

Reply:
updated.

Reply:
Was the push updated?